src/lib/fs.js

```diff
@@ -40,7 +39,7 @@ const rmdirRecursiveAsync = async (path) => {
   if (gte(NODE_VERSION, '14.14.0')) {
     return await rm(path, { force: true, recursive: true })
   }
-  await del(path, { force: true })
+  await rmdir(path, { force: true, recursive: true })
```

Comment:
Unlike `fs.promises.rm()` and `del`, `fs.promises.rmdir()` [does not have a `force` option](https://nodejs.org/api/fs.html#fspromisesrmdirpath-options). Running it against a non-existing file throws `ENOENT`, but only starting from Node 16.0.0

The `recursive` option with `fs.promises.rmdir()` [is deprecated](https://nodejs.org/api/fs.html#fspromisesrmdirpath-options). Starting from Node 16.0.0, using it prints an error message:

```
(node:24125) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
```

`fs.promises.rm()` requires Node 14.14.0.

We could possibly fix this by using a utility function along the lines of:

```js
const { promises: fs } = require('fs')

export const removeDir = function (path) {
  return fs.rm === undefined ? fs.rmdir(path, { recursive: true }) : fs.rm(path, { force: true, recursive: true })
}
```

However, I am wondering if we should just wait to drop Node 12, and use `fs.promises.rm()` then? What do you think?
Also cc @lukasholzer

Reply:
Correct me if I'm wrong, since there's code that's using `fs.rm` if Node.js is >= 14.14.0, then this path will be followed only on Node.js >= 12.20 and < 14.14.0.

I can't talk about `force`, I'll take your word for it :)

Reply:
Oh, I did not look at the context, only the line of code, and failed to notice this was already inside a test helper with a Node.js version condition :facepalm: Thanks for pointing this out.

This looks good then, except the `force` option can be removed since it does not exist. :+1:

Reply:
I removed `force` but I'm unsure about the change since the 2 code paths don't seem to be equivalent without `force`...

Reply:
I think the behavior should be the same, but I can't guarantee it. I'm fine with dropping this PR until Node.js 12.x support is dropped and you can clean up this file even more + do this in all of your project.